integration-tests/chopsticks/src/tests/xcm/assetHub/limitedReserveTransferAssetHubSpiritnet.test.ts

```diff
@@ -0,0 +1,69 @@
+import { test } from 'vitest'
+import { sendTransaction, withExpect } from '@acala-network/chopsticks-testing'
+
+import * as SpiritnetConfig from '../../../network/spiritnet.js'
+import * as AssetHubConfig from '../../../network/assetHub.js'
+import { DOT, keysAlice } from '../../../utils.js'
+import { spiritnetContext, assetHubContext } from '../../index.js'
+import { getSiblingLocation } from '../../../network/utils.js'
+import { createBlock, hexAddress, setStorage } from '../../utils.js'
+
+test('Limited Reserve Transfers from AH Account Alice -> Spiritnet Account Alice', async ({ expect }) => {
+	const { checkEvents, checkSystemEvents } = withExpect(expect)
+
+	// Assign alice some KSM
+	await setStorage(assetHubContext, {
+		...AssetHubConfig.assignDotTokensToAccounts([keysAlice.address]),
+		...AssetHubConfig.assignKSMtoAccounts([keysAlice.address]),
+	})
+
+	const spiritnetDestination = { V3: getSiblingLocation(SpiritnetConfig.paraId) }
+	const KSMAsset = { V3: [{ id: { Concrete: AssetHubConfig.KSMAssetLocation }, fun: { Fungible: DOT.toString() } }] }
```

Comment:
Why KSM tho?

Reply:
There is no DOT on rococo

Reply:
I just wanted to send a random asset. I decided for KSM. The asset description was the simplest, compared to the ERC-20 assets.

Reply:
Ah ok so this is any other asset than DOT?

Reply:
yes exactly